src/app.jsx

```diff
@@ -331,9 +331,16 @@ class Application extends React.Component {
         /* The following events do not need to trigger any state updates */
         case 'attach':
         case 'exec':
+        case 'exec_died':
         case 'export':
         case 'import':
         case 'init':
+        case 'kill':
+        case 'mount':
+        case 'prune':
```

Comment:
prune for example would have effect on the UI but I assume we get different event as well which is enough?

Reply:
Do you know what this event actually does/is?

Reply:
In my [discussion](https://github.com/containers/podman/issues/19124#issuecomment-1621297821) with @vrothberg he recommended to ignore internal transient events, but as I don't know what "prune" does, I can't judge if that's something internal, or an action that the user can do.

Reply:
It's [podman container prune](https://docs.podman.io/en/latest/markdown/podman-container-prune.1.html). Running these commands:
```
podman create localhost/test-busybox sleep infinity
podman container prune -f
```
does remove the container. `podman events` will show
```
2023-07-14 06:45:10.698039538 +0000 UTC container create a76bcda78923883a1abe5db42efbf0e4492d636888b3fd5b75529399d8844b76 (image=localhost/test-busybox:latest, name=gracious_dirac)
2023-07-14 06:45:10.683002845 +0000 UTC image pull 839899c307141ddc522d4018c208d46f5ac3050909a9a429775bb248ef5e6e42 localhost/test-busybox
2023-07-14 06:45:21.569598655 +0000 UTC container remove a76bcda78923883a1abe5db42efbf0e4492d636888b3fd5b75529399d8844b76 (image=localhost/test-busybox:latest, name=gracious_dirac)
```
i.e. it does not even get a `prune` event, but it does give a `remove` event.